Formula/pgcli.rb

```diff
@@ -1,3 +1,19 @@
+class PostgresRequirement < Requirement
+  fatal true
+
+  default_formula "postgresql"
+
+  satisfy { which "pg_config" }
+
+  def satisfied?
+    return_value = super
+    # Do not let Homebrew realize this requirement is satisfied by a formula.
+    # That way it won't try to update the formula.
+    @formula = nil
```

Comment:
You're relying on undocumented Homebrew internals here. Obviously you can do what you want but this is not a good idea if you wish this formula to remain stable without your intervention.